src/pages/home/HomePage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import logo from '../../logo.svg';
+
+export default class HomePage extends React.Component {
+    render() {
+        return (
+            <header className="App-header">
```

Comment:
We don't need this page anymore, our first page (on first access) will be the Onboarding page and after that will be the Forecast page.

Reply:
HomePage was created to simulate the Onboarding logic. It's required for the tests.